pub-critic/src/app/page.module.scss

```diff
@@ -102,6 +103,39 @@
           text-align: center;
         }
       }
+
+      .list {
+        width: 100%;
+        display: flex;
+        flex-direction: row;
+        align-items: flex-start;
+        justify-content: flex-start;
+        overflow-x: scroll;
+        margin-top: 20px;
+        > div {
```

Comment:
Would be better to use class selector than this way

Reply:
:white_check_mark: